kaldb/src/test/java/com/slack/kaldb/logstore/schema/ConvertFieldValueAndDuplicateTest.java

```diff
@@ -68,7 +68,7 @@ public void testListTypeInDocument() throws IOException {
                     Map.of("leaf2", "value2", "leaf21", 3, "nestedList", List.of(1)))));
 
     Document testDocument = docBuilder.fromMessage(message);
-    assertThat(testDocument.getFields().size()).isEqualTo(18);
+    assertThat(testDocument.getFields().size()).isEqualTo(23);
```

Comment:
All changes in this class are also related to the fact that now we default to STRING field. For String field we add a doc-value field as well. So all counts in this test need to be updated